hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -847,6 +847,9 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
 
             taker_slippage_adjustment_factor = Decimal("1") + self._slippage_buffer
             taker_balance = taker_balance_in_quote / (taker_price * taker_slippage_adjustment_factor)
+            if Decimal.is_nan(taker_price):
+                taker_balance = maker_balance
```

Comment:
Those are great ideas. Would you mind making a new PR that implements your suggestions? I can pull mine down.

Reply:
Hey, sorry I am only acting as a reviewer for this PR and do not intend on creating a new PR on the same issue.

Reply:
@andrewsiah please make the changes that @leastchaos proposed by April 30, otherwise this fix won't be included in the next release and will be rolled to the next month's release.